Gemfile

```
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby "~> 2.6"
+ruby "2.7.1"
```

```suggestion
ruby "~> 2.7.1"
```

We still probably want to keep the Pessimistic operator so we don't need to update the Gemfile for every ruby 2.7 patch release until 3.0 comes out.

---

That makes sense!